### DISCUSSION TITLE
Smaller font size? Maybe different font?

### DISCUSSION THREAD

**Discussion by fegyi001** [nvim-neo-tree/neo-tree.nvim#787] [Q&A] [answered]
I know this question must be silly, since nvim is in the terminal, but it would be so awesome if neo-tree could show smaller letters somehow. It does not need to be as big as my code font.

If there is any chance to adjust the font size (maybe the font itself?) that would be a huge improvement for me.

But... is there any chance? :)

Thanks

**Comment by cseickel** [comment, marked as answer]
This is not possible in a terminal. I think you can do it with some gui clients, but even there that would be a personal setting and not something controlled by the plugin.
